plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -749,6 +778,7 @@ private static ObjectWriteFunction longTimestampWithTimeZoneWriteFunction()
     {
         return ObjectWriteFunction.of(
                 LongTimestampWithTimeZone.class,
+                microsoft.sql.Types.DATETIMEOFFSET,
```

Comment:
nit: `import static microsoft.sql.Types.DATETIMEOFFSET;` ? (or is this how we disambiguate the `microsoft.sql` `Types`?